example/disasm/full.py

```diff
@@ -142,15 +142,13 @@
 
 
 # Generate dotty graph
-all_blocs = []
+all_blocs = BasicBlocks()
```

Comment:
I don't know if this is  intended, but I noticed that you switch to "blocks", in general. In some parts of this PR you leave it as "bloc". Especially in the code above and below, down to asmbloc.py.